src/components/PortfolioFilterBar/layouts/PortfolioSortEmptyDesktop.tsx

```
@@ -0,0 +1,19 @@
+import { useTranslation } from 'react-i18next';
+import { Select } from '@/components/core/form/Select/Select';
+import { SelectVariant } from '@/components/core/form/Select/Select.types';
+
+export const PortfolioSortEmptyDesktop = ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)